docs/setup-cluster/slurm/slurm-known-issues.rst

```diff
@@ -140,9 +140,8 @@ Some constraints are due to differences in behavior between Docker and Singulari
  Podman Known Issues
 *********************
 
--  Determined uses Podman in `rootless mode
-   <https://docs.podman.io/en/latest/markdown/podman.1.html#rootless-mode>`__. There are several
```

Comment:
This link seems to be working for me.

Reply:
The link-checker still freaks out about these, but that's fixed by adding the anchor to an anchor ignore-list. I'll add that into my PR. Should be fine.

Reply:
ok